components/__tests__/RadioButtonGroup-test.js

```diff
@@ -26,30 +26,30 @@ describe('RadioButtonGroup', () => {
 
       it('sets disabled attribute if disabled prop is set', () => {
         wrapper.setProps({ disabled: true });
-        expect(div.props().disabled).toEqual(true);
+        expect(wrapper.first('dev').props().disabled).toEqual(true);
```

Comment:
Is this a typo of `div`?

Reply:
I don't even think `first` tags in args so definitely my bad 😅